go/teams/invite_test.go

```diff
@@ -154,3 +155,65 @@ func TestKeybaseInviteMalformed(t *testing.T) {
 	require.NoError(t, err)
 	require.Len(t, team.GetActiveAndObsoleteInvites(), 0)
 }
+
+func TestMultiUseInviteChains(t *testing.T) {
+	team, _ := runUnitFromFilename(t, "multi_use_invite.json")
+
+	state := &team.chain().inner
+	require.Len(t, state.ActiveInvites, 1)
+
+	var inviteID keybase1.TeamInviteID
+	var invite keybase1.TeamInvite
+	for inviteID, invite = range state.ActiveInvites {
+		break // grab first invite
+	}
+
+	require.Equal(t, inviteID, invite.Id)
+	require.Nil(t, invite.ExpireAfterTime)
+	require.NotNil(t, invite.ExpireAfterUses)
+
+	require.Len(t, state.UsedInvites, 1)
+	usedInvitesForID, ok := state.UsedInvites[inviteID]
+	require.True(t, ok)
+	require.Len(t, usedInvitesForID, 3)
+
+	spew.Dump(state)
+
+	for _, usedInvitePair := range usedInvitesForID {
+		// Mostly check if UserLog pointed at by usedInvitePair exists
+		// (otherwise crash on map/list access).
+		ulog := state.UserLog[usedInvitePair.Uv][usedInvitePair.LogPoint]
+		require.Equal(t, ulog.Role, invite.Role)
+	}
+}
+
+func TestMultiInviteBadLinks(t *testing.T) {
+	fus, tcs, cleanup := setupNTests(t, 1)
+	defer cleanup()
+
+	_ = fus
```

Comment:
not actually a test yet. I was trying to figure out how to (ab)use `precheckLinksToPost` to test error conditions in parser